packages/client/src/runtime/core/engines/binary/BinaryEngine.ts

```diff
@@ -86,6 +86,7 @@ export class BinaryEngine extends Engine<undefined> {
   private stderrLogs = ''
   private currentRequestPromise?: any
   private platformPromise?: Promise<Platform>
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
   private platform?: Platform | string
```

Comment:
Perhaps the intent here was to have both autocompletion and freeform strings?
```suggestion
  private platform?: Platform | String
```

Reply:
Thanks for the suggestion, that does work for this case here.

Reply:
Actually that explodes https://github.com/prisma/prisma/actions/runs/6669272870/job/18126674298?pr=20165

Reply:
so I reverted to ignoring it